data/clients.json

```diff
@@ -636,7 +636,7 @@
         "last_renewed": "2020-12-10T19:00:55",
         "name": "txmmp",
         "platforms": [
-            "Unix"
+            "Linux"
```

Comment:
This looks like something that should not be part of this PR

Reply:
I only added this because the build validation was always failing with linter errors

Reply:
But I can remove when you want to fix the linter errors in a separate PR